### ISSUE TITLE
install

### ISSUE THREAD

**Issue by qthurier** [jexp/spoon-neo4j#3]
I didn't manage to make the pluggin work by adding the bookmarklet. Is Chrome supported? Can you develop the install?

**Comment by jexp** [comment]
Just drag the bookmark link to your bookmark bar. Yes chrome is supported.
Will look into a chrome extension when I find time.

**Comment by qthurier** [comment]
Yes, I've done that but I don't see the new features in the Neo4j browser.

**Comment by jexp** [comment]
Did you click it?

Von meinem iPhone gesendet

**Comment by qthurier** [comment]
.. Oups, that works now, thanks!

**Comment by ernestoe** [comment]
Is this working for the latest neo4j version?

**Comment by jexp** [comment]
@ernestoe unfortunately not as the new browser in 3.2.x is a complete rewrite and still quite a lot in flux.
I would have to redo all to make it work but haven't gotten the time right now.

**Comment by robomotic** [comment]
I am running version 3.0.12 and can't figure it out how to drag the link. I tried to drag the spoon.js url on the star symbol on the browser but it just opens the javascript source code.
Also dragged it on the Chrome bookmark bar but again is just placing the link there.
Not sure I fully understand.

**Comment by NaanProphet** [comment]
For what it's worth, I'm running Neo4j 3.4.1 and the browser-3x branch is working well for me in Safari (current commit is https://github.com/jexp/spoon-neo4j/commit/ff7edd2fab23bf84452f16f062ca036b8570083b).

**Comment by huyenqbi** [comment]
Any progress on this? I agree the instruction is very unclear.
